docs/solution-architecture/accessControlPolicies.md

```diff
@@ -112,6 +112,7 @@ various resources on which a user can perform actions:
 | starten_zaak                                         |                                                             ✅                                                             |             |                                                                    |           |
 | beheren                                              |                                                                                                                           |             |                                                                    |     ✅     |
 | zoeken                                               |                                                             ✅                                                             |             |                                                                    |           |
+| sjabloon_toewijzing                                  |                                                                                                                           |             |                                                                    |     ✅     |
```

Comment:
I think the word SmartDocuments should be in this policy otherwise people have no idea what this is about. A sjabloon can be anything. And I think the functionality is about linking SD templates (to zaakafhandelparemeters) right? I would suggest `smartdocuments_sjabloon_koppelen` as policy name.

Reply:
Changed to `smartdocuments_sjabloon_koppelen`